src/django_otp_webauthn/models.py

```diff
@@ -64,14 +65,6 @@ class Format(models.TextChoices):
         FIDO_U2F = "fido-u2f", "fido-u2f"
         NONE = "none", "none"
 
-    credential = models.OneToOneField(
```

Comment:
SInce this got moved and `__str__()` references self.credential, we may also want to specify just the type annotation for that field on this model if we want to run mypy on it regularly. I'm not going to fight with it right now, though, unless you'd rather solve for it immediately.

Reply:
I don't fancy picking a fight with the type checking system either. I'll leave that up to someone else as a future improvement.